### PR TITLE
[hostcfgd] Convert to Python 3; Add to sonic-host-services package

### PR DESCRIPTION
**Pull request by jleveque**
**- Why I did it**

To consolidate host services and install via packages instead of file-by-file, also as part of migrating all of SONiC to Python 3, as Python 2 is no longer supported.

**- How to verify it**

Ensure hostcfgd is installed and runs

**- Which release branch to backport (provide reason below if selected)**

- [ ] 201811
- [ ] 201911
- [ ] 202006

### CODE REVIEW

**Comment by jleveque** [comment]
Retest broadcom please

**Comment by jleveque** [comment]
Retest mellanox please

**Comment by jleveque** [comment]
Retest vsimage please

**Comment by jleveque** [comment]
Retest vsimage please

**Comment by jleveque** [comment]
Retest vsimage please

**Review by lguohan — APPROVED** [review]

**Comment by jleveque** [comment]
Retest vsimage please

**Comment by jleveque** [comment]
Retest vsimage please

**Comment by jleveque** [comment]
Retest vsimage please

**Comment by jleveque** [comment]
Retest vsimage please

**Comment by jleveque** [comment]
Retest vsimage please

**Comment by jleveque** [comment]
Retest vsimage please

**Comment by jleveque** [comment]
Retest vsimage please

**Comment by jleveque** [comment]
Retest vsimage please

**Comment by jleveque** [comment]
Retest vsimage please

**Comment by jleveque** [comment]
Retest vsimage please

**Comment by daall** [comment]
retest vsimage please

**Comment by jleveque** [comment]
This PR is blocked due to a Python 3 bug in swsssdk. @qiluo-msft is working on a fix.

File: src/sonic-host-services/scripts/hostcfgd [hunk 1]
```diff
@@ -306,7 +305,7 @@ class HostConfigDaemon:
 
     def update_all_feature_states(self):
         feature_table = self.config_db.get_table('FEATURE')
-        for feature_name in feature_table.keys():
+        for feature_name in list(feature_table.keys()):
```
Comment (qiluo-msft) [inline src/sonic-host-services/scripts/hostcfgd:308]:
```suggestion
        for feature_name in feature_table:
```

File: src/sonic-host-services/scripts/hostcfgd [hunk 2]
```diff
@@ -352,7 +351,7 @@ class HostConfigDaemon:
     def feature_state_handler(self, key, data):
         feature_name = key
         feature_table = self.config_db.get_table('FEATURE')
-        if feature_name not in feature_table.keys():
+        if feature_name not in list(feature_table.keys()):
```
Comment (qiluo-msft) [inline src/sonic-host-services/scripts/hostcfgd:354]:
```suggestion
        if feature_name not in feature_table:
```

Comment (jleveque) [reply src/sonic-host-services/scripts/hostcfgd]: Done

File: src/sonic-host-services/scripts/hostcfgd (hunk 1 above)
Comment (jleveque) [reply src/sonic-host-services/scripts/hostcfgd]: Done

**Review by qiluo-msft — DISMISSED** [review]

**Comment by jleveque** [comment]
Retest broadcom please

**Comment by jleveque** [comment]
Retest vs please

**Comment by jleveque** [comment]
Retest vsimage please

**Review by qiluo-msft — DISMISSED** [review]

**Comment by jleveque** [comment]
Retest vs please

**Comment by jleveque** [comment]
Retest vsimage please

**Comment by lguohan** [comment]
retest vsimage please

**Review by lguohan — DISMISSED** [review]

**Comment by jleveque** [comment]
Retest vsimage please

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**